src/Uno.Templates/content/unoapp/MyExtensionsApp._1.Skia.Gtk/MyExtensionsApp._1.Skia.Gtk.csproj

```
@@ -181,4 +181,10 @@
     <!--#endif-->
   </ItemGroup>
   <Import Project="..\MyExtensionsApp._1.Shared\base.props" />
+
+  <Target Name="ValidateOverrides" BeforeTargets="Restore;_CheckForUnsupportedTargetFramework" Condition="'$(OverrideTargetFramework)' != ''">
+    <Error
+        Text="OverrideTargetFramework set to '$(OverrideTargetFramework)' is invalid. Set OverrideTargetFramework to $baseTargetFramework$ or skip building this project (eg unload the project in Visual Studio)"
```

```suggestion
        Text="OverrideTargetFramework set to '$(OverrideTargetFramework)' is invalid. Set OverrideTargetFramework to $$(DotNetVersion) or skip building this project (eg unload the project in Visual Studio)"
```
do we want to provide the explicit baseTargetFramework? I would think we want to specify that you should use the DotNetVersion variable. Note the sample I have above uses the escape so the actual output shows to use the `$(DotNetVersion)`